prepare_cyrus.hs

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env stack
+-- stack runghc --resolver lts-8.14 --package turtle --install-ghc
```

Comment:
At the beginning I thought you would not accept commit with a script written in Haskell.
So I did not document it
To use it, it's very simple, you just need stack `sudo apt-get install haskell-stack`.
Then you just ./prepare_cyrus.hs, the first time it's going to take a lot of time because stack (a mix between nvm and npm) will download ghc